infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -157,6 +158,10 @@
                     + "              [default: TLSv1.1,TLSv1.2].\n"
                     + " --base-server-threads=<base_server_threads>                              "
                     + "              Number of threads dedicated for the base server.\n          "
+                    + " --log-size-quota=<max_log_size>                                          "
+                    + "              The max size of the log in bytes, if this limit is exceeded "
+                    + "              write requests will be rejected [default: "
+                    + Long.toString(Long.MAX_VALUE)+"].\n"
```

Comment:
"+" is redundant.

Reply:
If i remove either one on L164, compilation breaks. Which one are you talking about

Reply:
replaced it with -1, its cleaner now.